addons/hr_expense/models/hr_expense.py

```diff
@@ -288,9 +332,17 @@ def _check_product_uom_category(self):
                     expense=expense.name, product=expense.product_id.name,
                 ))
 
+    def _get_empty_expense_values(self, product):
+        return{
+            'name': _('product'),
```

Comment:
I don't see where in the specs it is asked to have this specific string.
Like already pointed out, this is very much likely to be
```suggestion
            'name': product.name,
```
or
```suggestion
            'name': product.display_name,
```

Reply:
I might have misunderstood that part - it's slide 9 - https://docs.google.com/presentation/d/1dKxs1Ag1leJ5a-iM9PSdFbVeWHmaoFUvK3NYlNet_Ok/edit#slide=id.g104139953b2_1_0

But what you are suggesting is way better, I will change it ^^